### ISSUE TITLE
Classic-style UI

### ISSUE THREAD

**Issue by OpenRift412** [fexed/Pinball-on-Android#43]
So I have a proposal for how the app could look a bit more polished and faithful to the original Windows version. I essentially took the UI elements and fitted them around a 9:16 aspect ratio, while also adjusting some graphics.

What do you think?

![new UI mockup](https://user-images.githubusercontent.com/29780788/177217589-58955d16-a9ab-4c2d-86f8-2575aded1786.png)

Here's the HUD graphic on its own if you want to use it.

![mobile UI graphic](https://user-images.githubusercontent.com/29780788/177217391-05ee1e34-0afa-4442-8ab3-afca01b820b3.png)

**Comment by fexed** [comment]
Will surely work on this!

**Comment by mherrmann3** [comment]
In case this suggestion causes a too large shrinking of the effective pinball area, here's an alternative: drop the left part with the "space cadet" (although funny) and reorder the two right boxes horizontally; the 'BALL X' field could go above the then-left box.

**Comment by OpenRift412** [comment]
> In case this suggestion causes a too large shrinking of the effective pinball area, here's an alternative: drop the left part with the "space cadet" (although funny) and reorder the two right boxes horizontally; the 'BALL X' field could go above the then-left box.

Honestly having that title graphic is half the charm of space cadet pinball if you ask me.

**Comment by fexed** [comment]
I agree with @OpenRift412 and will surely figure something out for preserving as much original GUI as possible
